integration/grid.py

```diff
@@ -0,0 +1,505 @@
+"""
```

Comment:
I expected this module to have something with the new `grid_manager` code, but ... it doesn't? Maybe this is a naming problem with the `grid_manager` rather than here, but it seems like you can make a grid without a manager.

Reply:
In general, yes you can make a grid without any grid-managers.

(I haven't re-familiarized myself with this yet, but at least some of this is "pure" refactoring because prior to this there wasn't really a case to make a bunch of different grids, so there was just one session-scoped one).